tests/integration/integration-test.sh

```diff
@@ -12,7 +12,7 @@ rm update
 echo 'Config set'
 
 echo 'Installing Connaisseur...'
-helm install connaisseur helm --atomic || { echo 'Failed to install Connaisseur'; exit 1; }
+make install || { echo 'Failed to install Connaisseur'; exit 1; }
```

Comment:
If you conciously choose a different installation method here that differs from the other ITs, I'd add a comment stating this explicitly

Reply:
But I like using both make and helm 👍